rdmo/projects/templates/projects/projects_pagination.html

```diff
@@ -4,17 +4,17 @@
     <div class="text-center">
         <ul class="pager">
             {% if page_obj.has_previous %}
-            <li class="previous"><a href="?page={{ page_obj.previous_page_number }}">{% trans 'Previous' %}</a></li>
+            <li class="previous"><a href="?page={{ page_obj.previous_page_number }}{% if querystring %}&amp;{{ querystring }}{% endif %}">{% trans 'Previous' %}</a></li>
             {% else %}
             <li class="previous disabled"><a href="">{% trans 'Previous' %}</a></li>
             {% endif %}
             <li>
                 {% blocktrans trimmed with number=page_obj.number num_pages=page_obj.paginator.num_pages %}
                 Page {{ number }} of {{ num_pages }}.
-                {% endblocktrans %}
+                {% endblocktrans %} ({{ projects|length }})
```

Comment:
Maybe I would add a "total" here. (which would need to be translated: `Seite 1 von 6. (20 insgesammt)`

Reply:
`{{ projects|length }}` is not always `20`, for the last page it might be less than `20` ! 😉

Reply:
at the bottom of the page it shows only the amount of the current page..